packages/cli/src/cli.mts

```diff
@@ -22,12 +25,12 @@ const run = async () => {
 
 	logger.info(`version ${version}`)
 
-	await checkAndUpdateSchemaVersion()
+	await checkAndUpdateSchemaVersion(options.project)
```

Comment:
can we add a validation here?
The string should end with `.json` and we should check if the file exists (except if it is a `setup` call).